### ISSUE TITLE
Extracting score map

### ISSUE THREAD

**Issue by acecreamu** [DrSleep/tensorflow-deeplab-resnet#187]
Hi Vladimir,

Could you give a pice of advice, how can I extract the score map (input before softmax function) from pretrained model? 

Thank you in advance

**Comment by DrSleep** [comment]
Using train.py as example, you can run sess.eval on raw_output. This would give a downsample score map, so if you want the full size one, you will need to do interpolation, too

**Comment by acecreamu** [comment]
I got good result just modifying inference.py. Thanks for answer, anyway.

**Comment by acecreamu** [comment]
Is it possible to extract deep features? I mean vector of encoded information in the middle of network.
If yes, which layer name should I specify in raw_output?  

Thank you in advance.
